doc/src/site/sphinx/deployment/sw_google_cloud_dataproc.rst

```diff
@@ -2,180 +2,135 @@ Running Sparkling Water on Google Cloud Dataproc
 ------------------------------------------------
 
 This section describes how to run Sparkling Water on `Google Cloud Dataproc <https://cloud.google.com/dataproc/docs/concepts/overview>`__. 
+It is meant to get you up and running with Sparkling Water on Google Cloud Dataproc as fast as possible so you can try it out.
+For further usage and productionizing some adjustments like `initialization actions <https://cloud.google.com/dataproc/docs/concepts/configuring-clusters/init-actions>`__ will be required.
+In this tutorial we will use Dataproc image version 2.0-debian10 which has Spark 3.1 and Scala 2.12.
 
-Perform the following steps to start Sparkling Water ``H2OContext`` on Cloud Dataproc. 
+1. Install the `Google Cloud SDK <https://cloud.google.com/sdk/docs/install>`__. Login to your account.
+2. Download `Sparkling water <http://h2o-release.s3.amazonaws.com/sparkling-water/spark-3.1/3.34.0.3-1-3.1/index.html>`__.
```

Comment:
Can we use a link to the latest version? `http://h2o-release.s3.amazonaws.com/sparkling-water/spark-3.1/latest.html`